app/ui/frontend/src/components/ResumeForm.vue

```diff
@@ -168,7 +169,17 @@ export default {
       this.update_html_in_modal(iframe_html, "modal-skills-ref-id");
     },
     getRank: function(index) {
-      return index + 1;
+      if (index == 0) {
```

Comment:
Javascript has switches! That would look better and be more extensible

https://www.w3schools.com/js/js_switch.asp

but again, not going to block a merge for it